plugins/mouse/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(PANEL_SOURCES
     ${QML_SOURCES}
 )
 
+configure_file(qmldir.in qmldir)
```

Comment:
Why do we need to configure the file? there is nothing to "configure" its a static file, its only for the "copy to build" dir you want, please set COPYONLY https://cmake.org/cmake/help/latest/command/configure_file.html